composer.json

```diff
@@ -44,6 +49,23 @@
 		},
 		"bamarni-bin": {
 			"target-directory": "bin/composer"
+		},
+		"changelogger": {
+			"changelog": "./changelog.txt",
+			"formatter": {
+				"filename": "bin/changelogger/WCAdminFormatter.php"
+			},
+			"versioning": "semvar",
```

Comment:
Should be `semver`

Reply:
Thank you @louwie17 👍 

Yeah, I think that's a good idea. I don't see us updating the major version anytime soon.

I'll address the changes in the next cooldown with a P2 post.

Reply:
Updated in https://github.com/Automattic/jetpack-changelogger/blob/master/src/AddCommand.php#L48

I need to confirm, but it looks like updating significances might not be possible at this time. The variable is [private](https://github.com/Automattic/jetpack-changelogger/blob/master/src/AddCommand.php#L48) and I don't see a config value that can be configured. ccing @anomiex to confirm.

Reply:
It's not possible right now, and I'd be skeptical of adding such a feature. Why do you want to? If you never make a "major" change, then just don't use it. If you are making "major" changes but aren't marking them as such, why not?